### DISCUSSION TITLE
部署到render时不时会提示Service Unhealthy

### DISCUSSION THREAD

**Discussion by Dreamkaka** [AlistGo/alist#4394] [Q&A] [answered]
部署在render上之后隔一段时间就会报错：This service is not ready to serve traffic. If you are the site owner, please check that your service is passing health checks.  重新清空部署缓存再次部署还是不能解决

日志如下：
<img width="702" alt="image" src="https://github.com/alist-org/alist/assets/97349507/fe6f75f7-7305-4cb0-ab09-4f6d14d25de0">

**Comment by John-Wong** [comment, marked as answer]
似乎是由于网站不活跃导致Render在约15分钟后自动关闭server，在使用uptime kuma等监控程序定时（如10分钟）发送请求保活后问题可得到解决。

**Reply by Dreamkaka** [reply]
谢谢解决了，感谢🙏
